extensions/ql-vscode/src/compare/interpreted-results.ts

```diff
@@ -34,37 +35,46 @@ export async function compareInterpretedResults(
     );
   }
 
-  const [fromResultSet, toResultSet, sourceLocationPrefix] = await Promise.all([
-    getInterpretedResults(
-      fromQuery.completedQuery.query.interpretedResultsPath,
-    ),
-    getInterpretedResults(toQuery.completedQuery.query.interpretedResultsPath),
-    database.getSourceLocationPrefix(cliServer),
-  ]);
-
-  if (!fromResultSet || !toResultSet) {
-    throw new Error(
-      "Could not find interpreted results for one or both queries.",
-    );
-  }
+  const { uniquePath1, uniquePath2, path, cleanup } = await cliServer.bqrsDiff(
+    fromQuery.completedQuery.query.resultsPath,
+    toQuery.completedQuery.query.resultsPath,
+    { retainResultSets: ["nodes", "edges", "subpaths"] },
```

Comment:
Better leave `retain-result-sets` to be the default, so there's only one place to update it if we add more result sets for problem/path-problem queries.